android/app/build.gradle

```diff
@@ -332,6 +332,10 @@ dependencies {
 
     // Plaid SDK
     implementation project(':react-native-plaid-link-sdk')
+
+    // Fixes a version conflict between airship and react-native-plaid-link-sdk
+    implementation "androidx.work:work-runtime-ktx:2.8.0"
```

Comment:
I remember Ryan saying about this. Is it something that can be resolved in the future? If so it might be worth including a bit more info, or an issue.